src/external/spglib.jl

```diff
@@ -116,7 +113,7 @@ function spglib_standardize_cell(lattice::AbstractArray{T}, atoms; correct_symme
     # Ask spglib to standardize the cell (i.e. find a cell, which fits the spglib conventions)
     num_atoms = ccall((:spg_standardize_cell, SPGLIB), Cint,
       (Ptr{Cdouble}, Ptr{Cdouble}, Ptr{Cint}, Cint, Cint, Cint, Cdouble),
-      spg_lattice, spg_positions, spg_numbers, length(spg_numbers), Cint(primitive),
+      copy(spg_lattice'), spg_positions, spg_numbers, length(spg_numbers), Cint(primitive),
```

Comment:
This won't work, because spglib modifies the lattice in-place.

Reply:
Good catch!